src/MissionEditor/MissionItemStatus.qml

```diff
@@ -17,31 +17,46 @@ import QGroundControl.Palette       1.0
 import QGroundControl               1.0
 
 Rectangle {
+    readonly property var       _activeVehicle:     QGroundControl.multiVehicleManager.activeVehicle
+
     property var    currentMissionItem          ///< Mission item to display status for
     property var    missionItems                ///< List of all available mission items
     property real   expandedWidth               ///< Width of control when expanded
 
     width:      _expanded ? expandedWidth : _collapsedWidth
-    height:     valueGrid.height + (_margins * 2)
+    height:     Math.max(valueGrid.height, valueMissionGrid.height) + (_margins * 2)
     radius:     ScreenTools.defaultFontPixelWidth * 0.5
     color:      qgcPal.window
     opacity:    0.80
     clip:       true
 
     readonly property real margins: ScreenTools.defaultFontPixelWidth
 
-    property real   _collapsedWidth:    valueGrid.width + (margins * 2)
+    property real   _collapsedWidth:    valueGrid.width + valueMissionGrid.width + (margins * 2)
```

Comment:
I think I'd rather keep the collapsed size to be smaller. This way it will still be workable on phones and not take up all the screen.